test/e2e/framework/azureidentity/azureidentity_helpers.go

```diff
@@ -9,6 +9,8 @@ import (
 	"io/ioutil"
 	"strings"
 
+	api "k8s.io/api/core/v1"
```

Comment:
naming convention in test/e2e:
```suggestion
	corev1 "k8s.io/api/core/v1"
```